ui_tests/test_ui_inputs.py

```diff
@@ -1617,9 +1619,37 @@ def test_error_in_subprocess(self):
                 patch.assert_called()
 
 
+@contextlib.contextmanager
+def _capture_logging_messages(logger, records):
+    """Capture logging messages and return them as a list.
+
+    Args:
+        logger (logging.Logger): A logger instance from python's logging system
+            to which log records should be captured.
+        records (list): A list to which log records should be appended.
+
+    Returns:
+        ``None``."""
+    log_queue = queue.Queue()
+    log_queue_handler = logging.handlers.QueueHandler(log_queue)
+    log_queue_handler.setLevel(logging.NOTSET)  # capture all logging
+    logger.addHandler(log_queue_handler)
+    yield
+
+    log_queue_handler.flush()
+    log_records = []
```

Comment:
Looks like you decided not to use this list

Reply:
Addressed in 344a8caf